app/controllers/accounts_controller.rb

```diff
@@ -15,7 +15,9 @@ def show
         render xml: AtomSerializer.render(AtomSerializer.new.feed(@account, @entries.to_a))
       end
 
-      format.activitystreams2
+      format.activitystreams2 do
+        headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
I've been trying to do that, but it doesn't seem to be working. I may just leave it out of this PR (it's not strictly necessary right now).